src/Sarif.Multitool/Rules/SARIF2004.OptimizeFileSize.cs

```diff
@@ -142,7 +218,6 @@ private void AnalyzeIdOnlyRules(Run run, string runPointer)
                     nameof(RuleResources.SARIF2004_OptimizeFileSize_Warning_EliminateIdOnlyRules_Text));
             }
         }
-
         private bool HasIdOnlyRules(string rulePointer)
```

Comment:
>p [](start = 8, length = 1)

Restore the blank line. (Our coding standard is "blank line after closing brace".)

Reply:
Oops

---
In reply to: [454705570](https://github.com/microsoft/sarif-sdk/pull/1995#discussion_r454705570) [](ancestors = 454705570)